OneCommunityGlobal/HGNRest
# Hotfix: Carl suggestion improvement

# Description
When a suggestion is made, we want the body of the email received by One community to include the name and email of the user making the suggestion. We also want a reply button which makes it easy for One community to reply to the specific user

Fixes # (bug list priority medium)
Or Implements # (WBS) 

## Related PRS (if any):
This backend PR is related to the [#1533](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1533) frontend PR.
To test this backendPR you need to checkout the [#1533](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1533) frontend PR.
…

## Main changes explained:
- Improved the email body received by one community when a user makes a suggestion …
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/volunteer/owner user
4. go to dashboard→ Suggestion Modal→ Make a Suggestion…
5. verify function “A” (feel free to include screenshot here)


## Note:
Kindly follow the instructions on the [frontend](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1533) PR
